scripts/linking.py

```diff
@@ -246,15 +235,15 @@ def create_tfidf_ann_index(model_path: str, umls_concept_list: List[str]) -> Non
 def load_tfidf_ann_index(model_path: str):
     # `S` for Search. This controls performance at query time. Maximum recommended value is 2000.
     # It makes the query slow without significant gain in recall.
-    efS = 100
+    efS = 1000
```

Comment:
Just curious, why increasing `efS` back again ?

Reply:
Oh sorry, I was just playing around. Once we move it into the main codebase we can be more strict about stuff.